src/extensions/scratch3_vizblocks/index.js

```diff
@@ -67,12 +67,14 @@ class Scratch3VizBlocks {
          */
         this._penSkinId = -1;
 
+        // Initiate canvas and center of coordinate system for the line chart / dot plot
+        this._chartWidth = 360;
+        this._chartHeight = 200;
+
         // Shared variables
         this._xMarkers = 10;
         this._yMarkers = 5;
         this._interval = 36;
-        this._xCenter = -160;
-        this._yCenter = -100;
```

Comment:
We should still keep these 2 lines here, now there are no declarations for `this._xCenter` and `this._yCenter` anymore.